### PR TITLE
[FEAT] EbayListboxButton: Adding missing tabIndex functionality from marko component

### PR DESCRIPTION
**Pull request by marcial199545**
Adding missing functionality to have dynamic `tabIndex` with value `0`when list is open and `-1` when it is closed.
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-listbox/index.marko#:~:text=tabindex%3D(option.disabled%20%3F%20%2D1%20%3A%20option.tabindex)

### CODE REVIEW

**Comment by darkwebdev** [comment]
Hello! Thanks for the PR! I'm trying to understand which problem are we solving here?
[Skin](https://opensource.ebay.com/skin/#listbox-button) always has `tabIndex=0`
[Marko](https://opensource.ebay.com/ebayui-core/?path=/story/buttons-ebay-listbox-button--standard) switches between 0 and -1
But is there a difference from a11y perspective? I thought that if menu is hidden then tabIndex doesn't matter...

File: src/ebay-listbox-button/listbox-button.tsx [hunk 1]
```diff
@@ -235,7 +244,7 @@ const ListboxButton: FC<EbayListboxButtonProps> = ({
                     <div
                         className="listbox-button__options"
                         role="listbox"
-                        tabIndex={0}
+                        tabIndex={listContainerTabIndex}
```
Comment (darkwebdev) [inline src/ebay-listbox-button/listbox-button.tsx:247]:
```suggestion
                        tabIndex={expanded ? 0 : -1}
```
and you don't need any more changes from above

**Review by darkwebdev — CHANGES_REQUESTED** [review]

**Comment by marcial199545** [comment]
retest

**Comment by marcial199545** [comment]
closing PR, to reopen it with changes proposed by [Timur Manyanov](https://github.com/darkwebdev)

**Pull request closed without merging**